javascript/modules/client/README.md

```diff
@@ -1663,6 +1758,74 @@ const configAction = client.encoding.updatePluginSettingsAction(
 
 ## Action decoders
 
+### Decode action grant permission
+
+```ts
+import { Client, Context, IPermissionParams, DaoPermissions } from "@aragon/sdk-client";
+
+const context: Context = new Context(params);
+const client: Client = new Client(context);
+
+const data: Uint8Array =  new Uint8Array([12, ..., 56])
+
+const grantParams = client.decoding.grantAction(
+  data
+);
+console.log(grantParams);
+/*
+{
+  who: "0x1234567890...",
+  where: "0x1234567890...",
+  permission: "0x1234567890...", // DaoPermission
```

Comment:
Shouldn't this be `"UPGRADE_PERMISSION"` ?

Reply:
It is, but the `console.log` will show it like this, but in the code the type will be `daoPermission` and if you try to equal it to `daoPermission.UPGRADE_PERMISSION` it will return `true`